ReactAndroid/src/main/res/devsupport/values/strings.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
   <string name="catalyst_reloadjs" project="catalyst" translatable="false">Reload JS</string>
-  <string name="catalyst_debugjs" project="catalyst" translatable="false">Debug in Chrome</string>
-  <string name="catalyst_debugjs_off" project="catalyst" translatable="false">Stop Chrome Debugging</string>
+  <string name="catalyst_debugjs" project="catalyst" translatable="false">Debug JS Remotely</string>
```

Comment:
I'm not sure "Remotely" add clarity here, especially since Chrome is running *locally*. Can you keep just "Debug JS", "Stop JS Debugging"?